src/plugins/controls/public/control_group/editor/data_control_editor_tools.ts

```diff
@@ -36,20 +44,13 @@ const doubleLinkFields = (dataView: DataView) => {
   return fieldRegistry;
 };
 
-export const loadFieldRegistryFromDataViewId = async (
-  dataViewId: string
+const loadFieldRegistryFromDataViewId = async (
```

Comment:
nit - How about renaming loadFieldRegistryFromDataViewId -> loadFieldRegistryFromDataView?

Reply:
Oh yup, good catch! I changed it to take the data view instead of the ID so that I could get memoization to work, and I forgot to update the name to reflect this change 👍